BDArmory/Control/MissileFire.cs

```diff
@@ -6143,8 +6143,17 @@ public void SendTargetDataToMissile(MissileBase ml, bool clearHeat = true)
                 if (BDArmorySettings.DEBUG_MISSILES)
                     Debug.Log($"[BDArmory.MissileData]: targetInfo sent for {ml.targetVessel.Vessel.GetName()}");
             }
+            else if (ml.TargetingMode == MissileBase.TargetingModes.Heat && ml.hasDataLink && !heatTarget.exists)
+            {
+                if (vesselRadarData)
+                {
+                    ml.vrd = vesselRadarData;
+                    if (vesselRadarData.locked) ml.targetVessel = vesselRadarData.lockedTargetData.targetData.vessel.gameObject.GetComponent<TargetInfo>();
+                }
+            }
             else if (ml.TargetingMode == MissileBase.TargetingModes.Radar)
             {
+                if (ml.bullDog) ml.hasDataLink = false;
```

Comment:
bulldog has previously been set to false if no datalink; what is the purpose of this? You're essentially stating that datalink will always be false.

Reply:
bulldog is a button (ln 777 to 786 on MissileLauncher and ln 287 on MissileBase) that creates a toggle between the BDA normal radar blindfire and Datalink Blindfire